tests/main/retry-tool/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: smoke test for the retry test tool
```

Comment:
This is really nice, but, we are currently using the main test suite also for testing our tools, I think should be usefull to use a different test suite for that, in fact we dont need most of the preparation for this. @zyga, What do you think?

Reply:
I was thinking about a suite that doesn't need to build snapd, that would be a nice improvement from current system. Note that this is not new: mountinfo-tool and version-tool are similarly tested. If anything I'd like to explore changing this in a separate PR.

Reply:
It's great you have a test for this ❤️ ; I agree it's would be nice to move it elsewhere, but that can be done in a followup indeed.